### PR TITLE
Seo updates

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/288
https://github.com/PureStake/moonbeam-docs/pull/293
https://github.com/PureStake/moonbeam-docs/pull/297

Skipped the files that are touched by other pending PR's

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

**Comment by hyd628** [comment]
@eshaben I skipped the files that are touched by other pending PR's

File: learn/platform/tokens.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: 代币
-description: 和所有的智能合约平台一样，Moonbeam Network也需要发行实用性代币让生态有效运转。发行在Polkadot平台上的代币为Glimmer（GLMR），在Kusama上的代币称作Moonriver（MOVR）。
+title: 实用代币
+description: 每个Moonbeam网络都需要一个实用代币才能运行 - Polkadot上Moonbeam的Glimmer(GLMR)和Kusama上Moonriver的Moonriver(MVR)。
 ---
 
 # 概览
```
Comment (eshaben) [inline learn/platform/tokens.md:6]: line 33 should use this updated URL /builders/get-started/networks/moonbase/#get-tokens/

File: node-operators/networks/collators/faq.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: FAQ
+title: 收集人常见问题
 description: 关于成为收集人和收集人活动的一些常见问题以及在Moonbeam上运行和操作收集人节点时的注意事项
 ---
```
Comment (eshaben) [inline node-operators/networks/collators/faq.md:4]: line 174 should use this updated URL /tokens/manage/identity/

File: node-operators/networks/run-a-node/overview.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 概览
+title: 运行节点
 description: 了解为Moonbeam网络运行平行链全节点以便您拥有自己的RPC端点或生成区块的所有必要细节
 ---
```
Comment (eshaben) [inline node-operators/networks/run-a-node/overview.md:5]: line 93 should use this latest URL /builders/build/eth-api/debug-trace

File: tokens/connect/ledger.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 使用Ledger
+title: 使用Ledger硬件钱包与Moonbeam交互
```
Comment (eshaben) [inline tokens/connect/ledger.md:2]: shouldn't include Moonbeam... should just be Connect & Use Ledger

File: tokens/connect/ledger.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
...
 description: 本教程将引导您如何利用Moonbeam兼容以太坊的特性，使用Ledger硬件钱包在Moonbeam内签署交易。
 ---
```
Comment (eshaben) [inline tokens/connect/ledger.md:5]: line 120 should use this updated URL /builders/get-started/networks/moonbase/#get-tokens

File: tokens/connect/mathwallet.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 使用MathWallet
+title: 连接MathWallet
 description: 通过此教程学习如何将Mathwallet（浏览器版）钱包连接至Moonbeam。
 ---
```
Comment (eshaben) [inline tokens/connect/mathwallet.md:5]: line 14 should use updated URL /builders/build/eth-api/dev-env/remix/

Comment (eshaben) [reply tokens/connect/mathwallet.md:5]: and line 104

File: tokens/connect/nifty.md [hunk 7]
```diff
@@ -69,7 +69,7 @@ description: 此教程将会带您了解如何连接Nifty钱包，一个基于
 
 ## 使用Nifty钱包 {: #using-nifty-wallet }
 
-Nifty钱包为Web3工具的提供者，如[Remix](/builders/tools/remix/)。通过将Nifty钱包连接至Moonbeam网络之一，您可以如同在MetaMask上部署合约般部署合约，并且通过Nifty签署交易。本教程会使用Moonbase Alpha测试网，但也适用于Moonbeam和Moonriver。
+Nifty钱包为Web3工具的提供者，如[Remix](/builders/build/eth-api/dev-env/remix/)。通过将Nifty钱包连接至Moonbeam网络之一，您可以如同在MetaMask上部署合约般部署合约，并且通过Nifty签署交易。本教程会使用Moonbase Alpha测试网，但也适用于Moonbeam和Moonriver。
```
Comment (eshaben) [inline tokens/connect/nifty.md:72]: line 78 should use the latest URL /builders/get-started/networks/moonbase/#get-tokens/

File: tokens/connect/trezor.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
-title: 使用Trezor
-description: 通过本教程学习如何利用Moonbeam兼容以太坊的特性，使用Trezor硬件钱包在Moonbeam内签署交易
+title: 如何连接和使用Trezor
+description: 通过本教程学习如何利用Moonbeam兼容以太坊的特性，连接和使用Trezor硬件钱包在Moonbeam内签署交易
 ---
```
Comment (eshaben) [inline tokens/connect/trezor.md:5]: line 87 should use this updated URL /builders/get-started/networks/moonbase/#get-tokens/

File: tokens/manage/multisig-safe.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Moonbeam多签钱包
+title: Moonbeam Safe多签钱包
 description: 学习如何通过Moonbeam Safe多签钱包使用和管理资金。在Moonbeam上创建一个新的多重签名钱包并接收和发送Token（以及ERC-20 Token）至Safe钱包。
 ---
```
Comment (eshaben) [inline tokens/manage/multisig-safe.md:5]: line 29 should use this updated URL /builders/get-started/networks/moonbase/#get-tokens/

File: tokens/manage/proxy-accounts.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 ---
-title: 代理账户
-description: 学习如何在基于Moonbeam的网络设置代理账户
+title: 设置代理账户
+description: 了解如何在基于Moonbeam的网络上设置代理帐户，以便您可以将钱包安全保存在冷存储中。
 ---
```
Comment (eshaben) [inline tokens/manage/proxy-accounts.md:5]: line 24  should use this updated URL /builders/get-started/networks/moonbase/#get-tokens/

File: tokens/staking/stake.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 ---
-title: 如何进行质押
-description: 本教程将展示如何通过委托收集人在Moonbeam质押Token
+title: 如何质押您的MOVR和GLMR代币
+description: 了解如何通过委托收集人在Moonbeam上质押代币并获得奖励。
 ---
```
Comment (eshaben) [inline tokens/staking/stake.md:5]: line 293 should use this updated URL /learn/features/staking/#reward-distribution

**Review by eshaben — CHANGES_REQUESTED** [review]
This was a tough one to review, so I'm sure it wasn't easy to create 😅 

All of the index.md files from this PR still need to be translated: https://github.com/PureStake/moonbeam-docs/pull/293/files

Minor changes in this PR https://github.com/PureStake/moonbeam-docs/pull/297/files# are missing for the following pages:
- tokens/manage/identity.md
- builders/integrations/analytics/dapplooker.md (Not sure if what already exists is enough, but please double check it)

A lot of comments were about this URL /builders/get-started/moonbase/#get-tokens/ not being updated to /builders/get-started/networks/moonbase/#get-tokens/

**Comment by hyd628** [comment]
@eshaben builders/integrations/analytics/dapplooker.md is being translated by CryptoDiscover on Trello (i havent made a jira ticket for it).

File: builders/build/substrate-api/index.md [hunk 12]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Substrate API
+description:
```
Comment (eshaben) [inline builders/build/substrate-api/index.md:3]: this one's missing... English version is:

```
Learn how to interact with the Substrate API when developing on Moonbeam, including how to use the Polkadot.js API for querying Moonbeam data.
```

File: builders/integrations/oracles/razor-network.md [hunk 13]
```diff
@@ -128,18 +128,12 @@ interface Razor {
...
 
-![Razor check price](/images/builders/integrations/oracles/razor/razor-demo-2.png)
-
-## 与联系我们 {: #contact-us } 
-
-如果您对在您的项目中实施Razor Network Oracle或任何其他Moonbeam相关主题有任何反馈，请随时通过我们的官方开发 [Discord 服务器](https://discord.com/invite/PfpUATX) 与我们联系。
-
---8<-- 'text/disclaimers/third-party-content.md'
\ No newline at end of file
+![Razor check price](/images/builders/integrations/oracles/razor/razor-demo-2.png)
```
Comment (eshaben) [inline builders/integrations/oracles/razor-network.md:139]:
```suggestion
![Razor check price](/images/builders/integrations/oracles/razor/razor-demo-2.png)

--8<-- 'text/disclaimers/third-party-content.md'
```

File: learn/platform/tokens.md [hunk 14]
```diff
@@ -1,6 +1,6 @@
 ---
-title: 代币
-description: 和所有的智能合约平台一样，Moonbeam Network也需要发行实用性代币让生态有效运转。发行在Polkadot平台上的代币为Glimmer（GLMR），在Kusama上的代币称作Moonriver（MOVR）。
+title: 实用代币
+description: 每个Moonbeam网络都需要一个实用代币才能运行 - Polkadot上Moonbeam的Glimmer(GLMR)和Kusama上Moonriver的Moonriver(MVR)。
```
Comment (eshaben) [inline learn/platform/tokens.md:3]:
```suggestion
description: 每个Moonbeam网络都需要一个实用代币才能运行 - Polkadot上Moonbeam的Glimmer(GLMR)和Kusama上Moonriver的Moonriver(MOVR)。
```

**Review by eshaben — CHANGES_REQUESTED** [review]
just one last thing!

**Comment by hyd628** [comment]
OK fixed it. Thanks @eshaben .

**Review by eshaben — APPROVED** [review]
Huge effort! Nice job!! 🙌 

🚀 🚢  ✨

**Pull request merged**